app/models/peoplefinder/concerns/completion.rb

```diff
@@ -16,23 +16,16 @@ module Peoplefinder::Concerns::Completion
     ]
 
     def completion_score_fields
-      fields = COMPLETION_SCORE_FIELDS
-      if no_phone
-        fields -= [:primary_phone_number]
-      end
-      fields
+      COMPLETION_SCORE_FIELDS
```

Comment:
Do we need the constant end this getter method both? I guess it was useful, but now we could maybe just keep the getter? The constant doesn't seem to be used anywhere else..